app/views/account/users/posts/update_gallery.js.erb

```diff
@@ -0,0 +1,9 @@
+$('#gallery-modal').modal('show');
```

Comment:
what do you think about CoffeeScript in js views? :)

Reply:
i think it is not 'coffee', it's just bottstrap-jquery methods, or I don't understand this